tests/testthat/test-use-tutorial.R

```diff
@@ -0,0 +1,39 @@
+context("test-use-tutorial")
+
+test_that("use_tutorial() requires a `name`", {
```

Comment:
I think you can combine this with the next test into something like "use_tutorial() checks its inputs"